### PR TITLE
Update saddle mainnet pools

### PR DESCRIPTION
**Pull request by cece-z**
## Description

Remove two outdated saddle pools:
* stables(Stablecoin): 0x3911f80530595fbd01ab1516ab61255d75aeb066
* bitcoins(BTC): 0x4f6a43ad7cba042606decaca730d4ce0a57ac62e

Add two new V2 pools:
* stablesV2: 0xaCb83E0633d6605c5001e2Ab59EF3C745547C8C7
* bitcoinsV2: 0xdf3309771d2BF82cb2B6C56F9f5365C8bD97c4f2

## Checklist:

-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [ ] Update documentation as needed.
-   [ ] Add new entries to the relevant CHANGELOG.jsons.

### CODE REVIEW

File: packages/asset-swapper/CHANGELOG.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 [
+    {
+        "version": "16.58.0",
+        "changes": [
+            {
+                "note": "Update Saddle pools on Mainnet",
+                "pr": 450
+    },
```
Comment (kh-chang) [inline packages/asset-swapper/CHANGELOG.json:8]: we haven't released 16.57.4 so I think we can move it under 16.57.4

File: .gitignore [hunk 2]
```diff
@@ -180,3 +180,4 @@ packages/*/docs/README.md
 
 # the snapshot that gets built for migrations sure does have a ton of files
 packages/migrations/0x_ganache_snapshot*
+.tool-versions
```
Comment (kh-chang) [inline .gitignore:183]: nit: is this needed?

**Review by kh-chang — APPROVED** [review]
LGTM

File: .gitignore (hunk 2 above)
Comment (cece-z) [reply .gitignore]: removed

**Review by kh-chang — APPROVED** [review]

**Pull request merged**